npm.php

```diff
@@ -22,3 +22,15 @@
     }
     run("cd {{release_path}} && {{bin/npm}} install");
 });
+
+desc('Install npm packages');
+task('npm:install_local', function () {
```

Comment:
I wonder about naming... wouldn't `npm:local:install` be better in general? We could then use that sort of thing as a standard :)